smoker/server/plugins/__init__.py

```diff
@@ -71,18 +74,18 @@ def stop(self, blocking=True):
         self.stopping = True
 
         # Trigger stop of all plugins
-        for plugin in self.plugins.values():
+        for plugin in list(self.plugins.values()):
```

Comment:
Is the explicit `list` call needed here (and also in other places)? `for plugin in <iterator>` should work just fine as well, right?